scripts/lp-bugs-to-gh-issues.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+
+from launchpadlib.launchpad import Launchpad
+from github import Github
+
+
+class GithubApi:
```

Comment:
This whole class looks like a whole load of overengineering - it looks to me like Github gives you everything you need:

``` python3
class GithubApi():
   # ... 30ish more lines

github_token = os.getenv("GITHUB_TOKEN")
github_repo = "canonical-web-and-design/maas-ui"
github_session = GithubApi()
github_session.set_env(github_token, github_repo)

# ...

github_session.create_issue(bug.title, issue_body)
```

Should be simply:

``` python3
github = Github(os.getenv("GITHUB_TOKEN"))
repo = github.get_repo("canonical-web-and-design/maas-ui")
repo.create_issue(bug.title, issue_body)
```

Removing about 35 lines of code.

Reply:
Awesome, much smaller :+1: